theta/include/theta_sketch.hpp

```diff
@@ -471,8 +471,14 @@ class wrapped_compact_theta_sketch_alloc : public base_theta_sketch_alloc<Alloca
 };
 
 template<typename Allocator>
-class wrapped_compact_theta_sketch_alloc<Allocator>::const_iterator: public std::iterator<std::input_iterator_tag, uint64_t> {
+class wrapped_compact_theta_sketch_alloc<Allocator>::const_iterator {
 public:
+  using iterator_category = std::input_iterator_tag;
+  using value_type = uint64_t;
+  using difference_type = void;
+  using pointer = uint64_t*;
+  using reference = uint64_t;
```

Comment:
const

Reply:
gcc complains (warning) about `const` here in the return type.

Reply:
right, let't not use const when returning by value